core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -520,6 +520,8 @@ object SparkSubmit {
       }
       if (args.principal != null) {
         require(args.keytab != null, "Keytab must be specified when the keytab is specified")
```

Comment:
I know this line wasn't changed in this PR but since we are in this file anyways could we maybe cleanup this error message (I think it is meant to say  "Keytab must be specified when the kerberos principal is specified"